osquery/sql/virtual_table.cpp

```diff
@@ -1059,7 +1059,7 @@ Status attachTableInternal(const std::string& name,
                            const std::string& statement,
                            const SQLiteDBInstanceRef& instance,
                            bool is_extension) {
-  if (SQLiteDBManager::isDisabled(name)) {
+  if (SQLiteDBManager::isDisabled(name) or !SQLiteDBManager::isEnabled(name)) {
```

Comment:
```suggestion
  if (SQLiteDBManager::isDisabled(name) || !SQLiteDBManager::isEnabled(name)) {
```

Reply:
This is not supported on MSVC and in general it's not the code style we use.

Reply:
Thanks I would not have found it on my own.